doc/usecases/plugins/spec/create_array_specification.md

```diff
@@ -0,0 +1,102 @@
+# Use Case: Create array specification for an application
+
+## Summary
+
+- **Scope:** `spec`
+- **Level:** Developer Goal
+- **Actors:** Application, User
+- **Brief:**
+  - Creating a specification for a REST application which needs communication to a mail server and message broker.
+
+## Scenarios
+
+- **Precondition:** kdb is installed.
+- **Main success scenario:**
+  - Specification is written user or system-wide.
```

Comment:
Please add more details about "is written", as it is it doesn't necessarily contain arrays. Verb after "system-wide" missing?